### DISCUSSION TITLE
help - can find element on my pc but not on other pc

### DISCUSSION THREAD

**Discussion by orbar1** [ultrafunkamsterdam/undetected-chromedriver#1378] [General]
Not sure if it's anything to do with the website's protection but would like to read some of your suggestions.
My issue is that I use this specific xpath to grab product's title:
(//ry-wishlist-product)[1]//div[@class="title my-3"]

thing is that my bot's .py and .exe i make of the .py are working great on my PC, but when i move the .exe to a VM or other laptop, the element isn't being detected for some reason. 
if i run inspect in the bot's browser, i can find this element so im not sure what is this behaviour, any idea would be greatly appreciated!

**Comment by patmann03** [comment]
did you compare the XPATH of both PC's to see what the difference is?  Many sites build themselves dynamically so the XPATH is not consistently in the same location.  

For example, my xpath for mine changes location.  I have to find a specific keyword, then traverse back up the tree then check for another value.  I'm using F strings so the first text it's finding is based on a variable: p

`"//*[@id='root']//*[text()='{p}']//parent::div//parent::div//parent::div//*[text()='Available']"`

**Reply by orbar1** [reply]
hey thank you :)
may i ask wdym if i compared the xpath?
the xpath i posted works on both PCs when the bot opens the website so i cant understand this behaviour

**Reply by patmann03** [reply]
So you're able to open the developer tools and the xpath is the same on both machines, but running the bot with the exe on another machine it doesn't work?

**Reply by orbar1** [reply]
on my pc it recognizes both items and sends me an email, on other PCs it goes to the "except" and prints no elements were found. same code, same exe, and same xpath works on both PCs when i enter it in the search line in the inspect.
never encountered it before :/

**Reply by patmann03** [reply]
Can you run the code in debug mode on the VM vs. using the executable?  I've pretty much stopped creating executable with python as I've always had issues with it.  Windows and other Anti-Virus's tend to flag them and quarantine them.

**Reply by orbar1** [reply]
i will have to install all the code dependecies and stuff but sure ill check the .py itself with debug mode :)
will update.
